index.html

```diff
@@ -24,6 +24,22 @@
     />
   </head>
   <body>
+    <div id="mobile_menu_container" class="mobile_menu hide">
```

Comment:
- [ ] Please make sure that the JS linters are setup correctly, as per [this](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/7_mobile_menu.md#:~:text=For%20this%20project%2C%20you%20should%20add%20the%20JavaScript%20%2B%20HTML%20%2B%20CSS%20linter.) requirement.